content/develop/reference/protocol-spec.md

```diff
@@ -530,6 +531,54 @@ The first element is a key, followed by the corresponding value, then the next k
 `key1, value1, key2, value2, ...`.
 {{% /alert %}}
 
+<a name="attribute-reply"></a>
+
+### Attributes
+
+The attribute type is exactly like the Map type, but instead of a `%` character as the first byte, the `|` byte is used. Attributes describe a dictionary exactly like the Map type. However the client should not consider such a dictionary part of the reply, but as auxiliary data that is used in order to augment the reply.
```

Comment:
Might be clearer to use "character" for the content and "byte" for the position consistently:
```suggestion
The attribute type is exactly like the Map type, but instead of a `%` character as the first byte, the `|` character is used. Attributes describe a dictionary exactly like the Map type. However the client should not consider such a dictionary part of the reply, but as auxiliary data that augments the reply.
```